### DISCUSSION TITLE
Frontend framework → shadcn/ui?

### DISCUSSION THREAD

**Discussion by achaido** [Open-Sustainability-Index/open-sustainability-index-backend#16] [Ideas]
I'm ponder if we should use use https://ui.shadcn.com/ as front-end framework? It feels like the graphs, tables etc are easier to implement than the ones used currently? Just an idea / suggestion / question.

**Comment by tomsoderlund** [comment]
I'd say it's quite low ROI on such a project; it's a pretty big task, and I wouldn't say Material UI (MUI) is such a big problem(?)*.

*Related: Although, Material UI (MUI) is what made us go from Remix.js to Next.js on this project and thus split OSI into separate backend and frontend → I'd prefer to have it in one project/repo. Long-term, that is… :-)
